modules/juce_graphics/geometry/juce_Point.h

```diff
@@ -25,6 +25,10 @@
 */
 
 #pragma once
+#if defined(_MSC_VER) && _MSC_VER < 1910
+#pragma push_macro("constexpr")
+#define constexpr /*constexpr*/

```

Comment:
I was under the impression that VS2013 has partial support for `constexpr`, so this code should be throwing errors at worst, warnings at best, no?